tsconfig.json

```diff
@@ -1,13 +1,13 @@
 {
   "compilerOptions": {
     "module": "Node16",
-    "target": "ES2020",
+    "target": "ES2022",
```

Comment:
One thing we should be careful about is that we want to support all platforms out of the box (Node.JS, browser, React Native). I'm not 100% react native supports ES2022.

If it's just for error cause we might be able to:
- define a custom zodError property on ValidationError
- in the toString of ValidationError do not log zodError. This way the zod error can still be accessed when needed but doesn't clutter the logs?


Then we can keep ES2020 for now to be on the safe side?

Reply:
Good idea! Let's do it this way

Reply:
I did that in my last commit using `Object.defineProperty` with `enumerable` set to false :
- [`54d7ad8` (#19)](https://github.com/openwallet-foundation-labs/oid4vc-ts/pull/19/commits/54d7ad8345542820bc7302feb518b88cd8be0a28#diff-965ee41279b5391872f12256e42d9b8e9bf87ff7d0c4ab7f8603caf654981c90R108)

Thanks to this, `zodError` will not be displayed in the terminal but will remain accessible, as in this test :
- [`54d7ad8` (#19)](https://github.com/openwallet-foundation-labs/oid4vc-ts/pull/19/commits/54d7ad8345542820bc7302feb518b88cd8be0a28#diff-c14efa3627e3dd5a13d637214ae3cd54639f65fca04b0165ba2fffe9300f3a75R38)